jupyterlab/src/index.ts

```diff
@@ -60,7 +60,7 @@ function registerQSharpLanguage(
     },
     {
       token: "keyword",
-      regex: String.raw`(let|set|w\/|new|not|and|or|use|borrow|using|borrowing|newtype|mutable)\b`,
+      regex: String.raw`(let|set|w\/|\<-|not|and|or|use|borrow|mutable)\b`,
```

Comment:
This still has the same word boundary issue that you already fixed in the tmLanguage file.

Reply:
Made some tweaks that I think address this, will be adding more operators soon...